client/my-sites/post-share/index.jsx

```diff
@@ -221,7 +225,10 @@ class PostShare extends Component {
 	}
 
 	renderUpgradeToGetSchedulingNudge() {
-		if ( this.props.hasRepublicizeSchedulingFeature ) {
+		if (
+			this.props.hasRepublicizeSchedulingFeature ||
+			! isEnabled( 'publicize-scheduling' )
```

Comment:
it isn't needed since the flag is controlled into the constants. https://github.com/Automattic/wp-calypso/blob/master/client/lib/plans/constants.js#L267